Livecheckables/rogue.rb

```diff
@@ -0,0 +1,4 @@
+class Rogue
+  livecheck :url   => "https://src.fedoraproject.org/repo/pkgs/rogue/",
+            :regex => /href=.*?rogue-v?(\d+(?:\.\d+)+)\.t/
```

Comment:
```suggestion
            :regex => /href=.*?rogue-?v?(\d+(?:\.\d+)+)(?:-src)?\.t/
```

The archives in this index use one of two formatting styles:

* `rogue-5.4.4.tar.gz`
* `rogue5.4.2-src.tar.gz`

There's always a chance that the next version could revert back to the older format (or use a mix of the two), so it's better to handle them both (as in the above suggestion).